nss-day-cohort-60/gamer-rater-v1
# Added auth/register functions to API and added supporting routes

# Description

Added an auth module to the views package with a `login_user` and a `register_user` function in it to create the appropriate rows in the database on register. Login was straight copy/pasta from LevelUp.

## Steps to test

1. Clone the awesome client repo that Sydney made for us
2. `npm i`
3. `npm start`
4. Register a new user and verify that token is in response
5. Log out again
6. Log in with credentials and verify that same token is in reponse
